src/Features/Core/Portable/NavigateTo/NavigateToSearcher.cs

```diff
@@ -148,18 +183,6 @@ private async Task SearchAllProjectsAsync(bool isFullyLoaded, CancellationToken
         /// </summary>
         private ImmutableArray<ImmutableArray<Project>> GetOrderedProjectsToProcess()
         {
-            // If we're only searching the current doc, we don't need to examine anything else but that.
```

Comment:
no need to special case this.  we only ever call this when searching all projects.